### PR TITLE
make capability-compare-function understand `WRITE_PROPERTIES`

### PR DESCRIPTION
**Pull request by haakonvt**

### CODE REVIEW

File: tests/tests_unit/test_data_classes/test_capabilities.py [hunk 1]
```diff
@@ -247,6 +250,38 @@ def test_load__action_does_not_exist(self, acl_cls_name: str, bad_action: str, d
 
         assert Capability.load(dumped, allow_unknown=True)
 
+    @pytest.mark.parametrize(
+        "has_write_allscope, has_write_props_allscope, has_write_in_same_scope",
+        itertools.product(*itertools.repeat([True, False], 3)),
+    )
+    def test_load_data_model_instances__with_write_properties(
```
Comment (erlendvollset) [inline tests/tests_unit/test_data_classes/test_capabilities.py:257]: Some comments explaining what's going on/being tested here would be good, been staring at it for 4 minutes and I still don't really think I get it 😬

**Review by erlendvollset — COMMENTED** [review]
Don't have enough context on this compatibility-comparator thingy to give a good review, so just some nitpicking and request for comments for now!

File: tests/tests_unit/test_data_classes/test_capabilities.py [hunk 2]
```diff
@@ -247,6 +250,38 @@ def test_load__action_does_not_exist(self, acl_cls_name: str, bad_action: str, d
 
         assert Capability.load(dumped, allow_unknown=True)
 
+    @pytest.mark.parametrize(
+        "has_write_allscope, has_write_props_allscope, has_write_in_same_scope",
+        itertools.product(*itertools.repeat([True, False], 3)),
```
[the remark this answers is not in the document]
Comment (haakonvt) [reply tests/tests_unit/test_data_classes/test_capabilities.py:255]: Huh, had no idea you could use `parametrize` to create combinations, that is brilliant 👌

**Review by erlendvollset — APPROVED** [review]
Perfect! 💯 Much easier to understand now!

**Pull request merged**